pkg/cli/root.go

```diff
@@ -33,9 +35,53 @@ var rootCmd = func() cli.Command {
 		Name:    "jvsctl",
 		Version: version.HumanVersion,
 		Commands: map[string]cli.CommandFactory{
+			"api": func() cli.Command {
```

Comment:
I'm open to other names instead of subcommands. `ui-server` and `public-key-server` and `cert-rotation-server` felt bad. I figured we might want `jvsctl rotation invoke` in the future for manual rotation or something, so I elected to do namespaced subcommands for everything.

Reply:
The existing names in the PR are fine to me. One nit: should they be `jvsctl api serve` rather than `server`?

Unrelated, I wonder what about the existing CLI names. Should they be

`jvsctl token create` and `jvsctl token validate`